astroplan/core.py

```diff
@@ -158,6 +157,52 @@ def __init__(self, location=None, timezone='UTC', name=None, latitude=None,
             raise TypeError('timezone keyword should be a string, or an '
                             'instance of datetime.tzinfo')
 
+    def astropy_to_local_time(self, astropy_time):
+        '''
+        Convert the `~astropy.time.Time` object ``astropy_time`` to a
+        localized `~datetime.datetime` object.
+
+        Parameters
+        ----------
+        time : `~astropy.time.Time`
+
+        Returns
+        -------
+        `~datetime.datetime`
+            Localized datetime, where the timezone of the datetime is
+            set by the ``timezone`` keyword argument of the
+            `~astroplan.Observer` constructor.
+        '''
+        # Convert astropy.time.Time to a UTC localized datetime (aware)
+        utc_datetime = pytz.utc.localize(astropy_time.utc.datetime)
+        # Convert UTC to local timezone
+        return self.timezone.normalize(utc_datetime)
+
+    def local_to_astropy_time(self, date_time):
+        '''
+        Convert the `~datetime.datetime` object ``date_time`` to a
+        `~astropy.time.Time` object.
+
+        If the ``date_time`` is naive, the implied timezone is the one set by
+        the ``timezone`` keyword argument of the `~astroplan.Observer`
```

Comment:
To be clear here you might instead say "the `timezone` structure of `self`" or something like that. That is, the user might have changed it since they constructed the object, so what really matters is the *current* value.